humminbird.cc

```diff
@@ -875,7 +875,7 @@ humminbird_rte_tail(const route_head* rte)
     be_write32(&humrte->time, humrte->time);
 
     QString name = mkshort(rtename_sh, rte->rte_name);
-    strncpy(humrte->name, CSTR(name), sizeof(humrte->name));
+    strncpy(humrte->name, CSTR(name), sizeof(humrte->name)-1);
```

Comment:
I can beleive this made some tool happy, but it is unclear to me if the format really uses a NULL terminator on a fixed length string, it may just be zero filled.  At least this change is consistent with L737.